WebCryptoAPI/idlharness.https.any.js

```diff
@@ -0,0 +1,14 @@
+// META: global=window,worker
+// META: script=/resources/WebIDLParser.js
+// META: script=/resources/idlharness.js
+
+promise_test(async() => {
+  const text = await (await fetch('/interfaces/WebCryptoAPI.idl')).text();
+  const idl_array = new IdlArray();
+  idl_array.add_untested_idls("[Global] interface Window { };");
+  idl_array.add_untested_idls("interface ArrayBuffer {};");
+  idl_array.add_untested_idls("interface ArrayBufferView {};");
```

Comment:
While you're here... you can avoid having explicit "mock" idl by fetching `/interfaces/html.idl` and using

    idl_array.add_dependency_idls(html);

Reply:
done